src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -135,13 +136,27 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return 1;
         }
 
+        $outputFormat = $input->getOption('output-format');
```

Comment:
Can you check that we are indeed using `--format` most of the time in the framework and deprecate `output-format` in favor of `format`?